tdvt/tdvt/tdvt.py

```diff
@@ -492,7 +497,7 @@ def init():
     return parser, ds_reg, args
 
 def is_test(args):
-    return args.command == 'run' or args.command == 'run-pattern'
+    return args.command == 'run' or args.command == 'run-pattern' or args.command == 'run-file'
```

Comment:
If this line feels like it's getting long we could use something like

```suggestion
    return args.command in ['run', 'run-pattern', 'run-file']
```

Reply:
fixed